commons/error_messages.py

```diff
@@ -51,14 +51,9 @@
                                "operation: This copy request is illegal because it is trying to "
                                "copy an object to itself without changing the object's metadata, "
                                "storage class, website redirect location or encryption attributes.")
-RGW_ERR_CPY_IF_MATCH_FALSE  = ("An error occurred (PreconditionFailed) when calling the "
-                               "CopyObject operation: Unknown")
-RGW_ERR_CPY_IF_NONE_MATCH_FALSE  = ("An error occurred (304) when calling the CopyObject "
-                                    "operation: Not Modified")
-RGW_ERR_CPY_IF_MODIFIED_SINCE_FALSE = ("An error occurred (304) when calling the CopyObject "
-                                       "operation: Not Modified")
-RGW_ERR_CPY_IF_UNMODIFIED_SINCE_FALSE = ("An error occurred (PreconditionFailed) when calling the "
-                                         "CopyObject operation: Unknown")
+RGW_ERR_CPY_IF_COND_FALSE  = ("An error occurred (PreconditionFailed) when calling the "
+                               "CopyObject operation: At least one of the pre-conditions you "
```

Comment:
this is the only error message for any pre-condition failure in copy object as per discussion with the dev